sdk/src/types/block/slot/index.rs

```diff
@@ -46,12 +46,17 @@ pub struct SlotIndex(pub u32);
 
 impl SlotIndex {
     /// Gets the [`EpochIndex`] of this slot.
-    pub fn to_epoch_index(self, slots_per_epoch_exponent: u8) -> EpochIndex {
-        EpochIndex::from_slot_index(self, slots_per_epoch_exponent)
+    pub fn to_epoch_index(self, genesis_slot: impl Into<Self>, slots_per_epoch_exponent: u8) -> EpochIndex {
+        EpochIndex::from_slot_index(genesis_slot, self, slots_per_epoch_exponent)
```

Comment:
Do mind changing the order to the same than `to_epoch_index`? stressing me out

Reply:
Done